flopy/mbase.py

```diff
@@ -1782,12 +1783,22 @@ def run_model(
     cargs : str or list, optional, default None
         Additional command line arguments to pass to the executable.
         (Default is None)
+    custom_print: callable
+        Optional callbale for printing. It will replace the builtin
+        print function. This is useful for shorter prints or integration
+        into other systems such as GUIs.
+        default is None, i.e. use the builtion print
```

Comment:
Just some typos

* callbale -> callable
* builtion -> builtin

Maybe also "shorter prints" -> "shorter output"?

Reply:
@wpbonelli  Thanks for finding my typos.  I guess, I should use a spell checker in my editor. ;) 

I am not sure about the wording. "shorter output" seems a bit too generic. I would also associate something like file output with it. How about "shorter print output"?

Reply:
I think "shorter print output" makes sense, or it could be merged as is if you prefer the first wording. I am happy to fix typos and merge, unless you are on it already?

Reply:
I fixed the typos and pushed my changes. Now `ruff` died on the test file, which did not touch.